code_generator/GdkCodeGenerator/Templates/UnityGameObjectComponentDispatcherGenerator.tt

```diff
@@ -60,21 +62,35 @@ namespace <#= qualifiedNamespace #>
                 return <#= unityComponentDefinition.Id #>;
             }
 
-            public override void InvokeOnComponentUpdateUserCallbacks(GameObjectDispatcherSystem gameObjectDispatcherSystem)
+            public override void InvokeOnAuthorityChangeLifecycleCallbacks(Dictionary<int, MonoBehaviourActivationManager> entityIndexToManagers)
```

Comment:
Potential drive by - these aren't callbacks. The things we are calling user callbacks are callbacks and the things we call lifecycle callbacks are just methods.